app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class IngredientsController < ApplicationController
+  def index
+    @ingredients = Ingredient.all
```

Comment:
So we would not want both in here. We started out with `.all` but then with the extension we would change it to `.alphabetical_name`. It is still giving us all of the ingredients.